cypress/integration/retention.js

```diff
@@ -20,14 +20,14 @@ describe('Retention', () => {
         cy.get('[data-attr=prop-val-0]').click()
         cy.get('[data-attr=retention-table').should('exist')
 
-        cy.get('.percentage-cell').last().click()
+        // cy.get('.percentage-cell').last().click()
```

Comment:
What's up with this? If it's not relevant any more I would remove.

Reply:
You're working on retention right? Are we getting rid of the table?

Reply:
The table should still be there 🤞 

Was there an issue with it? It may be that it's out of the viewport?

Reply:
It doesn't return any users or returns the wrong ones.

Reply:
Right, interesting, can you get a screenshot of what it's getting so I can check?

Reply:
I'm also fine to leave this as a followup tbh

Reply:
![image](https://user-images.githubusercontent.com/1727427/147934442-2a907d53-d93f-4528-bcb5-3fbf8d917543.png)
Just returns nothing basically, even in the chart itself.